app/vulncode_app/views.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.http import HttpResponse
+from django.shortcuts import render
+
+import vulncode_app.api_data as api
+
+def product(request, product):
+    """
+    Queries the cve-search api with just
+    a product name.
+    """
+    data = api.output_cve_id(name=product)
+    return HttpResponse(data)
+
+def product_ver(request, product, ver):
```

Comment:
I had primarily done it to sort of create a little differentiation, b/w what's being passed and what it is being checked to. But, your point is more valid. 😄